Dockerfile

```diff
@@ -1,21 +1,19 @@
-FROM python:3.8-alpine
+FROM python:3.10-slim-bullseye
```

Comment:
can we use 3.11, since it's out now?

Reply:
yes, I confirmed that unit test is passed.  But should I not change that of version?